### DISCUSSION TITLE
crash package rest api

### DISCUSSION THREAD

**Discussion by philippenc** [jaredhendrickson13/pfsense-api#580] [General]
hello,
i have a crash on my pfsense which has to be related to the rest api package. here it is
Crash report begins.  Anonymous machine information:

amd64
15.0-CURRENT
FreeBSD 15.0-CURRENT #0 plus-RELENG_24_03-n256311-e71f834dd81: Fri Apr 19 00:28:14 UTC 2024     root@freebsd:/var/jenkins/workspace/pfSense-Plus-snapshots-24_03-main/obj/amd64/Y4MAEJ2R/var/jenkins/workspace/pfSense-Plus-snapshots-24_03-main/sources/FreeBS

Crash report details:

PHP Errors:
[03-Oct-2024 11:00:30 Pacific/Noumea] PHP Fatal error:  Uncaught TypeError: RESTAPI\Caches\RESTAPIVersionReleasesCache::get_data_to_cache(): Return value must be of type array, null returned in /usr/local/pkg/RESTAPI/Caches/RESTAPIVersionReleasesCache.inc:27
Stack trace:
#0 /usr/local/pkg/RESTAPI/Core/Cache.inc(68): RESTAPI\Caches\RESTAPIVersionReleasesCache->get_data_to_cache()
#1 /usr/local/pkg/RESTAPI/.resources/scripts/manage.php(179): RESTAPI\Core\Cache->process()
#2 /usr/local/pkg/RESTAPI/.resources/scripts/manage.php(436): refresh_cache()
#3 {main}
  thrown in /usr/local/pkg/RESTAPI/Caches/RESTAPIVersionReleasesCache.inc on line 27

thank you

**Comment by jaredhendrickson13** [comment]
What version of the package are you using? This was reported in #543 and a fix was included in v2.1.3.

**Reply by philippenc** [reply]
v2.1.0, 
I am installing 2.1.3.
thanks
